src/__swaps__/utils/swaps.ts

```diff
@@ -21,7 +21,7 @@ import { ETH_ADDRESS, supportedNativeCurrencies } from '@/references';
 import { userAssetsStore } from '@/state/assets/userAssets';
 import { colors } from '@/styles';
 import { BigNumberish } from '@ethersproject/bignumber';
-import { CrosschainQuote, ETH_ADDRESS as ETH_ADDRESS_AGGREGATOR, Quote, QuoteParams, SwapType, WRAPPED_ASSET } from '@rainbow-me/swaps';
+import { CrosschainQuote, ETH_ADDRESS as ETH_ADDRESS_AGGREGATOR, Quote, QuoteParams, SwapType } from '@rainbow-me/swaps';
```

Comment:
can also remove imports of:
- `isLowerCaseMatchWorklet` on line 13
- `ETH_ADDRESS` on line 20
- SwapType on line 24